### ISSUE TITLE
[Corsair K55]: iCUE intercepts key presses from main keyboard after PC unlock

### ISSUE THREAD

**Issue by bharatmpg** [TaranVH/2nd-keyboard#104]
Tested with unmodifed scripts. Happens ocassionally after PC unlock not 100% repro. 

After I unlock my PC, iCUE starts wrapping key presses from my main keyboard with F24. If I run into this state the main keyboard is un-usable until I press a key on the K55.

I am suspecting a bug in iCUE. I would be cautious going with the iCUE approach as there is a possibility to execute macros unintentionally.

**Comment by TaranVH** [comment]
wait.... so what keyboard is your main keyboard?

**Comment by bharatmpg** [comment]
My main keyboard is a Microsoft Surface keyboard. 
https://www.microsoft.com/en-us/p/surface-keyboard/8r3rqvvflp4k

**Comment by bharatmpg** [comment]
Figured out the issue. The issue is with the "fingerprint reader" key in the surface keyboard. I noticed that the surface keyboard sends an F24 with I press the finger print reader key and sometimes the key gets stuck in pressed state and is not release. The AHK script intercepts all subsequent inputs coming from the main keyboard because the F24 key is not released.  Pressing a key on the macro keyboard fixes it because all keypresses on macro keyboard release the F24 key.

I wrapped my keys with F23 instead of F24 to work around the issue. 

BTW: Better to wrap keys with F24 instead of F23 because Win+F24 is shortcut to take a screenshot in  windows 10. With F23 wrapping i can use the windows modifier with other keys on the macro keyboard.
